packages/tests/src/api/server/logs.ts

```diff
@@ -193,6 +193,41 @@ describe('Test logs', function () {
       expect(logsString.includes('video 10')).to.be.true
       expect(logsString.includes('video 11')).to.be.false
     })
+
+    it('Should refuse to create logs if disabled', async function () {
+      this.timeout(100000)
+
+      await server.config.updateCustomSubConfig({
+        newConfig: {
+          instance: {
+            logs: {
+              auditLogs:{
+                enabled: false
+              }
+            }
+          }
+        }
+      })
+
+      await server.videos.upload({ attributes: { name: 'video 12' } })
+      await waitJobs([ server ])
```

Comment:
Just upload one video to test audit logs. Test 403 if disabled, but also the audit log was not filled when it was disabled (so re-enable the feature and check the audit log doesn't contain your upload)